pkg/junit/types_test.go

```diff
@@ -0,0 +1,22 @@
+package junit
+
+import (
+	"encoding/xml"
+	"testing"
+)
+
+const junitXML = `<testsuites>
+<testsuite tests="1" failures="0" time="1983" name="">
+<properties>
+<property name="go.version" value="go1.17.5 linux/amd64"/>
+</properties>
+<testcase classname="" name="TestUpgradeControlPlane" time="1983"/>
+</testsuite>
+</testsuites>`
+
+func Test_CanUnmarshalTestSuites(t *testing.T) {
+	suites := &TestSuites{}
```

Comment:
in order to get this test to run/pass, I needed an import of `"github.com/openshift/origin/pkg/test/ginkgo/junitapi"`